crates/forge/tests/e2e/diagnostics_and_plugins.rs

```diff
@@ -1,127 +1,128 @@
 
         error: Plugin diagnostic: Expected fork config must be of the form `url: <double quote string>, block_id: <snforge_std::BlockId>`.
-         --> {mock_tests_dir_path_str}/tests/test_fork.cairo:62:7
-        #[fork(url: "http://188.34.188.184:7070/rpc/v0_7", block_id: Hash(0x12345))]
+         --> [..]/tests/test_fork.cairo[..]
+        #[fork(url: "{node_rpc_url}", block_id: Hash(0x12345))]
               ^*******************************************************************^
 
         error: Plugin diagnostic: Expected fork config must be of the form `url: <double quote string>, block_id: <snforge_std::BlockId>`.
-         --> {mock_tests_dir_path_str}/tests/test_fork.cairo:68:7
-        #[fork(url: "http://188.34.188.184:7070/rpc/v0_7", block_id: Tag(Latest))]
+         --> [..]/tests/test_fork.cairo[..]
+        #[fork(url: "{node_rpc_url}", block_id: Tag(Latest))]
               ^*****************************************************************^
 
         error: Plugin diagnostic: Expected fork config must be of the form `url: <double quote string>, block_id: <snforge_std::BlockId>`.
-         --> {mock_tests_dir_path_str}/tests/test_fork.cairo:74:7
-        #[fork(url: "http://188.34.188.184:7070/rpc/v0_7", block_id: BlockWhat::Number(12345))]
+         --> [..]/tests/test_fork.cairo[..]
+        #[fork(url: "{node_rpc_url}", block_id: BlockWhat::Number(12345))]
               ^******************************************************************************^
 
         error: Plugin diagnostic: Expected fork config must be of the form `url: <double quote string>, block_id: <snforge_std::BlockId>`.
-         --> {mock_tests_dir_path_str}/tests/test_fork.cairo:80:7
-        #[fork(url: "http://188.34.188.184:7070/rpc/v0_7", block_id: Something::BlockId::Number(12345))]
+         --> [..]/tests/test_fork.cairo[..]
+        #[fork(url: "{node_rpc_url}", block_id: Something::BlockId::Number(12345))]
               ^***************************************************************************************^
 
         error: Plugin diagnostic: Expected panic must be of the form `expected: <tuple of felt252s and strings>` or `expected: "some string"` or `expected: <some felt252>`.
-         --> {mock_tests_dir_path_str}/tests/test_should_panic.cairo:2:15
+         --> [..]/tests/test_should_panic.cairo[..]
         #[should_panic(url: "https://test.com")]
                       ^***********************^
         Error: Failed to compile test artifact, for detailed information go through the logs above
         
         error: Plugin diagnostic: Expected fork config must be of the form `url: <double quote string>, block_id: <snforge_std::BlockId>`.
-         --> {mock_tests_dir_path_str}/tests/test_fork.cairo:86:7
-        #[fork(
-              ^
+         --> [..]/tests/test_fork.cairo[..]
+        #[fork(url: "http://188.34.188.184:7070/rpc/v0_7", block_id: BlockId::Tag(xddd::d00pa::hehe::BlockTag::Latest))]
```

Comment:
```suggestion
        #[fork(url: "{node_rpc_url}", block_id: BlockId::Tag(xddd::d00pa::hehe::BlockTag::Latest))]
```
should this url be here?

Reply:
Nope, missed that, thanks for noticing!